source/jquery.flot.time.js

```diff
@@ -284,7 +388,9 @@ API.txt for details.
 
         var step = tickSize * timeUnitSize[unit];
 
-        if (unit === "millisecond") {
+        if (unit === "microsecond") {
+            d.setMicroseconds(floorInBase(d.getMicroseconds(), tickSize));
+        } else if (unit === "millisecond") {
```

Comment:
What about a default else case instead?

Reply:
There was no default case before. What should this default to?

Reply:
There will be no default, as this is harvested from an initialized variable.